examples/api/package.json

```diff
@@ -12,24 +12,25 @@
     "build": "run-s copy-license sync-package-info build:*",
     "build:typescript": "tsc -b ./src/tsconfig.json",
     "watch": "npm-run-all copy-license sync-package-info --parallel watch:*",
-    "watch:server": "nodemon --inspect=0.0.0.0:9240 ./dist/index.js",
+    "watch:server": "nodemon --exec 'node -r dotenv/config' --inspect=0.0.0.0:9240 ./dist/index.js",
```

Comment:
Didn't know about the direct support. thanks :)
I would not include the `dotenv/config` in the `index.ts` since it shouldn't be used in production. The env vars will be either inside the docker container or delivered by the kubernetes deployment. If you do not include the module it will also not load `.env` files checked into git by mistake.